### PR TITLE
Fix conflict between `Hash#values_at` and "Parallel Assignment"

### PR DESCRIPTION
**Pull request by ydakuka**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2297,7 +2297,7 @@ Prefer the use of exceptions from the standard library over introducing new exce
 === Parallel Assignment [[parallel-assignment]]
 
 Avoid the use of parallel assignment for defining variables.
-Parallel assignment is allowed when it is the return of a method call, used with the splat operator, or when used to swap variable assignment.
+Parallel assignment is allowed when it is the return of a method call, used with the splat operator, or when used to swap variable assignment, or `Hash#values_at` and `Hash#fetch_values`.
```
Comment (ydakuka) [inline README.adoc:2300]: These built-in methods are very convenient, but there are many such methods.

Comment (bbatsov) [reply README.adoc:2300]: Those are method calls, so it's already covered in this sentence.

File: README.adoc [hunk 2]
```diff
@@ -4404,18 +4404,25 @@ batman.fetch(:powers, obtain_batman_powers) # obtain_batman_powers is an expensi
 batman.fetch(:powers) { obtain_batman_powers }
 ----
 
-=== `Hash#values_at` [[hash-values-at]]
+=== `Hash#values_at` [[hash-values-at-and-fetch-values]]
```
Comment (bbatsov) [inline README.adoc:4407]: You should check if something in RuboCop doesn't refer to this anchor before changing it.

Comment (ydakuka) [reply README.adoc]: Sure, there are no links to this anchor.

**Pull request merged**